be/src/util/compression/compression_context_pool_singletons.h

```diff
@@ -0,0 +1,129 @@
+// This file is made available under Elastic License 2.0.
```

Comment:
If we not changed, it's better to keep the original license without this header.

Reply:
i change this file a lot.